src/genie/libs/parser/iosxe/show_cts.py

```diff
@@ -149,3 +148,94 @@ def filter_lines(raw_output, remove_lines):
                 continue
         return cts_pacs_dict
 
+
+
+# =================================
+# Schema for:
+#  * 'show cts role-based counters'
+# =================================
+class ShowCtsRoleBasedCountersSchema(MetaParser):
+    """Schema for show cts role-based counters."""
+
+    schema = {
+        "cts_rb_count": {
+            int: {
+                "src_group": Or(str, int),
```

Comment:
I would recommend to use only `str` for `src_group` and `dst_group` if the value could be both. please change to just `str`.

other than this, schema looks good

Reply:
This has been updated

Reply:
confirmed